README.md

```diff
@@ -45,12 +43,22 @@
 
 ## Local Development
 
+Most of the following operations require minio client binary available locally at `./mc`
```

Comment:
Is it just for testing that a local instance of `mc` is required?
A system-wide `mc` installation should not be an issue, right?

Reply:
yes, you can use a system-wide mc